linetools/isgm/abssystem.py

```diff
@@ -351,6 +360,66 @@ def list_of_abslines(self):
         return [iline for component in self._components
                 for iline in component._abslines]
 
+    def measure_restew(self, spec=None, **kwargs):
+        """ Measure rest-frame EWs for lines in the AbsSystem
+        Parameters
+        ----------
+        spec : XSpectrum1D, optional
+        kwargs
+
+        Returns
+        -------
+
+        """
+        # Grab Lines
+        abs_lines = self.list_of_abslines()
+        # Loop
+        for iline in abs_lines:
+            # Fill in spec?
+            if spec is not None:
+                iline.analy['spec'] = spec
+            # Measure
+            iline.measure_restew(**kwargs)
+
+    def measure_aodm(self, spec=None, **kwargs):
+        """ Measure ADOM columns for the list of lines
+        Note: Components are *not* updated by default
+
+        Parameters
+        ----------
+        spec : XSpectrum1D, optional
+        kwargs
+
+        Returns
+        -------
+
+        """
+        # Grab Lines
+        abs_lines = self.list_of_abslines()
+        # Loop
+        for iline in abs_lines:
+            # Fill in spec?
+            if spec is not None:
+                iline.analy['spec'] = spec
+            # Measure
+            iline.measure_aodm(**kwargs)
+        #
+        print("You may now wish to update the component column densities")
```

Comment:
"You may now wish to update the component column densities. If so, please use `update_component_colm()`"